src/components/createElementComponent.test.tsx

```diff
@@ -10,6 +10,7 @@ import {
   PaymentRequestButtonElementComponent,
   CartElementComponent,
 } from '../types';
+import {PayButtonElement} from '..';
```

Comment:
Should we instead do import `PayButtonElementComponent` in the above import line?